readme.md

```diff
@@ -11,6 +11,12 @@ Don't know what a middleware is? [Check the CakePHP documentation](http://book.c
 This plugin is for CakePHP 3.4+.
 If your application existed before (<= CakePHP 3.3), please make sure it is adjusted to leverage middleware as described in the [docs](http://book.cakephp.org/3.0/en/controllers/middleware.html#adding-the-new-http-stack-to-an-existing-application).
 
+## What is it?
+This package is an Authentication middleware layer for authenticating requests coming into your application, using various authenticators and indentifiers.
+
+## What it is not
+The package only provides authentication and not authorization. It can be used to find out who someone is, but not what they're allowed to do. 
+
```

Comment:
It seems many users think this is a drop in replacement for core auth component. So it's better to add this clarification.